src/_store/root.js

```diff
@@ -35,7 +35,15 @@ export default class Root extends PureComponent {
         try {
             await tryAuth(token);
         } catch (e) {
-            actions.updateAppState('authorized', false);
+            await actions.updateAppState('authorized', false);
+            const errorMsg = e.message;
+            if (errorMsg.indexOf('you have excluded yourself until') > -1) {
```

Comment:
Likely need to catch by error type, since you will not have this string when translated :)